script.xbmcbackup/resources/lib/dropbox/auth.py

```diff
@@ -0,0 +1,862 @@
+# -*- coding: utf-8 -*-
```

Comment:
maybe the dropbox section can be turned into a script.module?

Reply:
Yes, in the description above I make mention of that fact that I do want to use the script.module.dropbox addon for this eventually. It is not currently at a version that supports Python3, I created a PR but still waiting. I can remove these in a future version once it's updated.

Reply:
Where is the PR? You mean against our repo or against the upstream version?